scilpy/tracking/tracking_field.py

```diff
@@ -27,10 +26,14 @@ class SphericalHarmonicField(object):
         Maximum angle (radians) between two steps.
     dipy_sphere: string
         Name of the DIPY sphere object to use for evaluating SH.
+    angle_maxima_detection: float
+        Angle used for peak extraction to check if a direction is maximal in
+        its neighbourhood.
     """
 
     def __init__(self, odf_dataset, basis, sf_threshold, sf_threshold_init,
-                 theta, dipy_sphere='symmetric724'):
+                 theta, dipy_sphere='symmetric724',
+                 angle_maxima_detection=np.pi / 16.):
```

Comment:
I'm not a fan of magic numbers. Why pi/16 ?

Reply:
Can you please write a sentence for me, Charles? I will add it.

Reply:
I'd rename it `min_separation_angle`. Here is my take on documenting it.
```
min_separation_angle: float, optional
    Minimum separation angle for peaks extraction. Used for deterministic
    tracking. A candidate direction is a maximum if its SF value is greater
    than all other SF values in its neighbourhood, where the neighbourhood
    includes all the sphere directions located at most `min_separation_angle`
    from the candidate direction.  
```

And sorry I made a mistake, it does not correspond to twice the minimum separation angle for peak extraction, it is exactly the minimum separation angle between two candidate peaks.